test/cag-extrude.js

```diff
@@ -47,9 +47,9 @@ test('CAG should extrudeInPlane', t => {
   t.is(s1.toPolygons().length, 66)
   var s2 = c2.extrudeInPlane('-X', 'Z', 10, {symmetrical: true})
   t.is(s2.toPolygons().length, 70)
-  var s3 = c3.extrudeInPlane('-Y', '-Z', 100, {symmetrical: false})
+  var s3 = c3.extrudeInPlane('-Y', '-Z', 100, {symmetrical: 'false'})
```

Comment:
is there some reason why boolean inputs fail ?

Reply:
The Boolean argument function supports both Boolean and strings. Actually, so does the float argument function.

This is just adding coverage.

Reply:
Wow nice catch & nice way to increase our coverage, well done ! :)